### PR TITLE
Resolve EX-7

### PR DESCRIPTION
**Pull request by maggietuttle**
This PR will add the source files for an empty 2D Unity project to the repository, along with an `Assets/Tests` folder which contains two example unit test suites, `SomeTests.cs` and `MoreTests.cs`. Each suite has two tests, and all are purposefully trivial to showcase the C# `Assert` class. `SomeTests::Test2FailsOnPurpose()`, as suggested by the name, is supposed to fail in order to showcase what a failing unit test looks like.

### CODE REVIEW

**Comment by maggietuttle** [comment]
This is a comment that isn't attached to a review.

**Review by maggietuttle — COMMENTED** [review]
Here I'm making a comment in the review section. You can do this by going to the "Files changed" tab on this PR. In that tab, you can look through each of the changed files, mark them as viewed, and then finally submit a review as either a comment, an approval, or a request for changes.

**Review by kayleenasser — APPROVED** [review]
Looks good to me! Nice work with the test scripts! We open the game using the Scene.unity file :)

**Review by TysonTopia — APPROVED** [review]
Looks Good and Explained well

File: ProjectSettings/NavMeshAreas.asset [hunk 1]
```diff
@@ -0,0 +1,93 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!126 &1
+NavMeshProjectSettings:
+  m_ObjectHideFlags: 0
+  serializedVersion: 2
+  areas:
```
Comment (AlanAyy) [inline ProjectSettings/NavMeshAreas.asset:7]: Looks a bit sus, should be fine though?

Comment (maggietuttle) [reply ProjectSettings/NavMeshAreas.asset:7]: Sus indeed

**Review by AlanAyy — APPROVED** [review]
LGTM!

File: ProjectSettings/NavMeshAreas.asset (hunk 1 above)
Comment (kayleenasser) [reply ProjectSettings/NavMeshAreas.asset:7]: Nah its fine

**Pull request merged**